cypress/integration/racingcar.spec.js

```diff
@@ -115,3 +178,46 @@ describe('예외 상황', () => {
       });
   });
 });
+
+describe('로딩', () => {
+  beforeEach(() => {
+    cy.visit('index.html');
+  });
+
+  it('자동차 경주 게임의 턴이 진행 될 때마다 1초의 텀을 두고 진행한다.', () => {
+    const inputString = '우디,꼬재';
+    const racingCount = '3';
+
+    cy.get(SELECTOR.$CAR_NAME_INPUT).type(inputString);
+    cy.get(SELECTOR.$CAR_NAME_BUTTON).click();
+
+    cy.get(SELECTOR.$RACING_COUNT_INPUT).type(racingCount);
+    cy.get(SELECTOR.$RACING_COUNT_BUTTON).click();
+
+    waitRepeatedly(
+      SELECTOR.$SPINNER,
+      'be.visible',
+      DELAY.RACING_PROGRESS,
+      racingCount
+    );
```

Comment:
waitRepeatedly을 호출할 때 넘기는 인자가 많아서 이게 어떤 값인지 다시 waitRepeatedly함수를 확인해야 하고, 인자를 입력하는 순서를 알아야 하는 부분때문에 아래와 같이 객체로 묶어서 키, 값으로 전달할 수 있게 하는게 어떨까 해요~ 
```suggestion
    waitRepeatedly({
      selector: SELECTOR.$SPINNER,
      type: 'be.visible',
      delay: DELAY.RACING_PROGRESS,
      repeatCount: racingCount
    });
```

```js
function waitRepeatedly({selector, type, delay, repeatCount})
```
받는 파라미터가 2개 이상일 때 객체로 묶어서 인자로 넘기고 함수에서 받을 때 전달 받은 객체를 구조분해해서 사용할 수 있게 리팩토링해볼 수 있겠네요.

Reply:
넵.. 전달되는 인자가 많으면 객체로 묶어주는 방법이 가독성 측면에서 좋은 것 같아요!! 감사합니다.🙂